docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,163 @@
+The Temporal Service manages critical tasks like making an API call or updating a database, by making sure they get executed, and then recording their completion.
+
+One of the biggest advantages of the Temporal Service is how it handles failures. 
+The Temporal Service maintains a meticulous log of every step in your Workflows. 
+By keeping a history of every step in your Workflow, it ensures that even if something goes wrong — your Workflow can continue from the last successful point. 
+The Temporal Service knows exactly where to resume without losing any work. 
+This saves you from having to write complex error-handling code or recovery mechanisms yourself. 
+
+You can run the Temporal Service on your own infrastructure or use Temporal Cloud, a managed service that handles operational overhead and offers scalability and expert support.
+
+![Describing a Temporal Application and Temporal Service - How It Works](/img/how-it-works/htw_application_service_server.png)
+
+
+## Workers
+The real strength of Temporal comes from the combination of your application and the Temporal Service. 
+A Worker is a component within your application and provided by the Temporal SDK. 
+Workers execute the actual tasks defined in your Workflows. The Temporal Service tracks and coordinates these Workers. 
+
+Whenever your application needs to perform a task, like sending a notification or calling an processing a payment, the Temporal Service steps in. 
+This process creates a clear, organized list of what needs to be done. 
+It hands off tasks to these Workers, making sure they’re done correctly and efficiently while it keeps track of their progress. 
+Workers which are part of your application then carry out the tasks and run your code. 
+This collaboration is crucial for building reliable, scalable, and durable applications. 
+You can run multiple Workers — often dozens, hundreds, or even thousands — to improve application performance and scalability. 
+
+![Describing a Temporal Application and Temporal Service and Workers - How It Works](/img/how-it-works/htw_application_service_workers.png)
+
+:::info 
+For a deep dive into how Temporal Service works, visit the [Temporal Encyclopedia page](/clusters).
+:::
+
+
+## Recovering from Failure
+
+In any complex system, failures are bound to happen, but how they are handled makes all the difference. 
+Temporal ensures that these failures don’t disrupt your application. 
+This reliability is achieved through **Durable Execution**.
+
+In a traditional non-Temporal application, if a crash occurs, the progress of your Workflow is often lost. 
+The application has no memory of what happened before the failure, requiring extensive error-handling and recovery code to resume. 
+This process is time-consuming and error-prone, making it hard to ensure reliability.
+
+With Temporal, if your Workflow crashes, it automatically recreates its pre-crash state and continues as if the failure never happened. 
```

Comment:
You've mentioned the points in this paragraph several times already, so it's kind of repetitive. Maybe it would be better to have an 'Overview' section and have the items in this paragraph as bullet points if you want to re-emphasize these points.

Reply:
https://www.notion.so/temporalio/Cribbed-from-Tom-2f7b42438c8549289ba6c0dc1edabd73

Grabbed some great words you can crib from @tomwheeler